internal/prompter/prompter.go

```diff
@@ -49,11 +50,20 @@ type surveyPrompter struct {
 	stderr    io.Writer
 }
 
+func Filter(filter, value string, index int) bool {
+	filter = text.RemoveDiacritics(strings.ToLower(filter))
```

Comment:
for this one, I went with the same behaviour as `ToLower`, which is by default applied to both filter and value. 

I think it's valid also for diacritics since you could type a name thinking the value contains diacritics whereas it does not?

No strong opinion, if you want to change this I can 👍